### PR TITLE
handlr: init at 0.5.0

### PR DESCRIPTION
**Pull request by mredaelli**
###### Motivation for this change

New interesting package

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mredaelli** [comment]
The tests fail, at least one for a permission error on a directory.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116790` at ee8b120d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/ddd790c30862c3d6159719a3fdfdcc9d">handlr</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `stdenv`.
Near pkgs/tools/misc/handlr/default.nix:1:8:
```
  |
1 | { lib, stdenv, rustPlatform, fetchFromGitHub }:
  |        ^
```
</li>
  </ul>
</details>

File: pkgs/tools/misc/handlr/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  src = fetchFromGitHub {
+    owner = "chmln";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1f4gmlqzgw1r8n0w9dr9lpsn94f2hlnak9bbq5xgf6jwgc9mwqzg";
+  };
+
+  cargoSha256 = "16d4dywwkgvvxw6ninrx87rqhx0whdq3yy01m27qjy4gz6z6ad8p";
+
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/handlr/default.nix:16]: Please add a comment why you disabled tests.

Comment (mredaelli) [reply pkgs/tools/misc/handlr/default.nix:17]: Done

**Comment by mweinelt** [comment]
Please, please, always investigate the reason why tests do not work. Handlr relies on `shared-mime-info` and a permissions problem during check phase is an indicator for improper HOME env var.

Fixed in #120474 

It also always makes sense to talk to the upstream developer. https://github.com/chmln/handlr/issues/35

**Comment by SuperSandro2000** [comment]
Yeah, I shouldn't let people get past me so easily with disabled tests. I try to remember that. Thanks for the reminder!

**Pull request merged**